updateParticipantData.json

```diff
@@ -407,5 +407,162 @@
 	"173836415.266600170.110349197": {
 		"dataType": "array",
 		"mustExist": false
+	},
+	"525972260": {
+		"values": [104430631, 353358909],
+		"dataType": "number",
+		"mustExist": false
+	},
+	"637153953": {
+		"dataType": "array",
+		"mustExist": false
+	}, 
+	"637153953[]": {
+		"dataType": "object",
+		"mustExist": false
+	}, 
+	"637153953[].345545422": {
+		"dataType": "ISO",
+		"mustExist": false,
+		"required": true
+	},
+	"637153953[].457270069": {
+		"dataType": "string",
+		"mustExist": false,
+		"maxLength": 800
+	},
+	"637153953[].740819233": {
+		"dataType": "object",
+		"mustExist": false
+	},
+	"637153953[].740819233.939782495": {
```

Comment:
is there a reason you're adding the brackets in the key value?

Reply:
Yep, this comes in as an array - we use this to validate the array data, and it's flexible because we can make a direct comparison from rule to array element regardless of the array element's index.

Process:
Take the incoming data, then flatten it to dot notation.
Then we have a data point like `637153953[0].345545422`
If the array has multiple objects, we'll also have
`637153953[1].345545422`, `637153953[2].345545422`, ...

Remove the index for a direct comparison between data and the rule, regardless of the array's size:
`const validationPath = path.replace(/\[\d+\]/g, '[]');`